api/nakadi-event-bus-api.yaml

```diff
@@ -1503,11 +1535,37 @@ definitions:
         type: string
         default: 'random'
 
+      compatibility_mode:
+        description: |
+          Compatibility modes provide a mean for event owners to evolve their schema, given changes
+          respect the semantics defined by this field.
+
+          It's designed to be flexible enough so that producers can evolve their schemas while not
+          inadvertently breaking existent consumers.
+
+          Once defined, the compatibility mode is fixed, since otherwise it would break a predefined contract,
+          jeopardizing the value of this feature.
+
+          List of available compatibility modes:
+
+          - 'deprecated': Schema changes are not allowed. This is used only by legacy event types. This option is not
```

Comment:
I think this could be called `fixed`. Deprecated means something different. I'm not sure how this will be implemented but it sounds like it will be a date check on the schema creation and something that a producer will never be allowed to set themselves. I'm wondering if it's actually needed. 

(gr)